### PR TITLE
Migrating URL unit tests to use networking v1 ingress

### LINKED ISSUE #4865: Upgrade url package unit tests to use networking v1 ingress by default
/kind task
Follow up of https://github.com/openshift/odo/issues/4592

### PR DESCRIPTION
**Pull request by mohammedzee1000**
**What type of PR is this?**
/kind task

**What does this PR do / why we need it**:
See topic

**Which issue(s) this PR fixes**:

Fixes #4865

**PR acceptance criteria**:

- [x] Unit test 

- [ ] Integration test 

- [ ] Documentation 

- [ ] Update changelog

- [x] I have read the [test guidelines](https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc)

**How to test changes / Special notes to the reviewer**:
unit tests in url package should not be broken

### CODE REVIEW

**Comment by mohammedzee1000** [comment]
psi windows timeout
/test psi-unit-test-windows

File: pkg/kclient/fake/ingress.go [hunk 1]
```diff
@@ -2,63 +2,74 @@ package fake
...
-				IngressSpecParams: generator.IngressSpecParams{
-					IngressDomain: "example-1.com",
-					ServiceName:   "example-1",
-					PortNumber:    intstr.FromInt(9090),
-				},
-			}),
+func GetKubernetesIngressListWithMultiple(componentName, appName string, networkingV1Supported, extensionV1Supported bool) *unions.KubernetesIngressList {
+	kil := unions.NewEmptyKubernetesIngressList()
+	ki1 := unions.NewKubernetesIngressFromParams(generator.IngressParams{
+		ObjectMeta: metav1.ObjectMeta{
```
Comment (mik-dass) [inline pkg/kclient/fake/ingress.go:19]: Please use more descriptive variable names

Comment (mohammedzee1000) [reply pkg/kclient/fake/ingress.go:19]: Fixing

**Comment by mohammedzee1000** [comment]
Unrelatted fail
/retest

**Review by mik-dass — APPROVED** [review]
/approve

**Pull request merged**